### PR TITLE
[CIS-616] Update README with more info about 3.x

### PR DESCRIPTION
**Pull request by VojtaStavik**
Users are now encouraged to start using v3 for their new projects.

### CODE REVIEW

**Review by nuno-vieira — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ The **StreamChatUI SDK**  is the official iOS SDK for [Stream Chat](https://gets
 
 ## Important ⚠️ 
 
-**This README refers to a beta version of `StreamChatUI` not suitable for production yet. The last stable version is [2.6](https://github.com/GetStream/stream-chat-swift/tree/main_v2/).**
+**We're currently in the final stage of releasing a new major version (3.0) of our SDKs.** Version 3.0 is already avaialable in public beta. We expect to release the production version during Q1/2021.
 
-**StreamChatUI** is a framework containing highly customizable UI components to build your chat UI with Stream chat using UIKit or SwiftUI. 
+If you want to integrate StreamChat SDKs into your app, you can already use version `3.0.beta.x`. However, please be aware that small API breaking changes can occure until a stable version is reached.
```
Comment (b-onc) [inline README.md:22]:
```suggestion
If you want to integrate StreamChat SDKs into your app, you can already use version `3.0-beta.x`. However, please be aware that small API breaking changes can occure until a stable version is reached.
```

**Review by b-onc — APPROVED** [review]
One small fix, :shipit:

**Comment by lambardi** [comment]
Very nice! LGTM

**Pull request merged**